### ISSUE TITLE
Wish: An option to only export rooms "visible in view"

### ISSUE THREAD

**Issue by evapeva** [Autodesk/revit-ifc#156]
**We're missing a possibility to select what rooms to export.**

Today, if we want to export only parts of the project file, for example only the first level, or only the objects in a certain workset, we can control this for model elements by using "Export only elements visible in view". We're missing the same possibility for rooms. It's all or nothing for rooms.

**Alternative 1**, hopefully quite simple to achieve:
Make Revit respect open/closed worksets when exporting. That way we could filter out the rooms to export/not export by placing them in different worksets. This is not a perfect workflow, but would be better than what we have today.

**Alternative 2**, requires changes to room behavior in Revit:
Make rooms visible in 3D views and make them follow "Export only elements visible in view". This would be the perfect workflow.

**Comment by AngelVelezSosa** [comment]
Currently, if you have a section box active, it will only export rooms inside (or touching the boundary of) the section box.  Do you need more control than that?

**Comment by AngelVelezSosa** [comment]
(And I do like Alternative 2).

**Comment by evapeva** [comment]
Yes, section box works in all cases where the area you want exported is square and tidy. Unfortunately that's not always the case. We sometimes have requests to only export one department etc. Department boundaries are most often not straight. (You konw, why make life easy?)

**Comment by ArtemHnatko** [comment]
This issue is also reported in Jira as "REVIT-240227" and is currently in the "Open" status.
